archunit-junit/junit4/src/test/java/com/tngtech/archunit/junit/internal/ArchUnitRunnerRunsMethodsTest.java

```diff
@@ -119,6 +121,18 @@ public void ignores_all_methods_in_classes_annotated_with_ArchIgnore() throws In
                 .contains(toBeIgnoredTwo);
     }
 
+    @Test
+    public void ignores_all_methods_in_base_classes_of_classes_annotated_with_ArchIgnore() throws InitializationError {
+        ArchUnitRunnerInternal runner = new ArchUnitRunnerInternal(IgnoredArchTestWithBaseClass.class);
```

Comment:
We could consistently use
```suggestion
        ArchUnitRunnerInternal runner = newRunnerFor(IgnoredArchTestWithBaseClass.class);
```
(eliminating the need to declare the `InitializationError`)